os/board/rtl8730e/src/component/wifi/inic/inic_ipc_host_api_ext.c

```diff
@@ -897,10 +897,19 @@ int wifi_csi_config(rtw_csi_action_parm_t *act_param)
 	int ret = 0;
 	u32 param_buf[1];
 
-	param_buf[0] = (u32)act_param;
-	DCache_Clean((u32)act_param, sizeof(rtw_csi_action_parm_t));
+	rtw_csi_action_parm_t *act_param_temp = (rtw_csi_action_parm_t *)rtw_zmalloc(sizeof(rtw_csi_action_parm_t));
```

Comment:
After this line, you do memcpy for entire buffer size. Do you need zmalloc? The zmalloc do malloc + memset with zero. `memset with zero` will be redundant.

Reply:
Agreed, zmalloc will not be necessary. I will make the changes, thanks